libraries/botframework-connector/src/auth/certificateAppCredentials.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @module botbuilder
```

Comment:
Not sure these are used anymore, and the other files in this folder use `botbuilder` instead of `botframework-connector`. Marking this as resolved and not going to do anything.